src/schema/broadband-signup-us.json

```diff
@@ -268,8 +272,8 @@
         "CreditCheckOrNull": {
             "oneOf": [{
                     "type": "null"
-                },
-                {
+                    },
```

Comment:
This change is not necessary, leads to the wrong indentation. Same for the next line.

Reply:
Yeah...the formatter from VSC...